pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -67,6 +67,7 @@ type BackingServiceSelector struct {
 	metav1.GroupVersionKind `json:",inline"`
 	ResourceRef             string `json:"resourceRef"`
 	// +optional
+	Id        string  `json:"id,omitempty"`
```

Comment:
Can we have a short description for the optional `id` field here as it has been provided for other optional fields as well.

Reply:
I'd prefer to have this change in a different PR.